src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -704,6 +704,17 @@ EncodedResults StaticLLMPipeline::generate(
         position_ids_data[0] = m_kvcache_desc.num_stored_tokens;
         attention_mask_data[m_kvcache_desc.num_stored_tokens - 1] = 1u;
 
+        // NB: Write KV-cache for the new token to the correct input position for the next iteration
```

Comment:
Comment should be changed

Reply:
@TolyaTalamanov Does set_tensor now support remote tensor data sharing across CPU and NPU?